### ISSUE TITLE
NUL files being written all over the place

### ISSUE THREAD

**Issue by victoriafrench** [rollbar/rollbar-php#399]
```
OS: MacOS 10.13.6
PHP: 7.2.7
using Apache in CGI Mode
```

In version `1.6.0` and `1.6.1` we are observing files called `NUL` being written all over the directory structure of our web site. This is not occurring with `1.5.3` and prior. We have confirmed that removing the `1.6.x` series stops the files from appearing and then re-adding them the problem returns.

It appears that any directory that has a php file read for a web request results in the file appearing, but we have not been able to confirm if that is exactly what is happening.

**Comment by victoriafrench** [comment]
trackback placeposition/alpaka.io#1586 (allows us to see the status of this issue on ours)

**Comment by Dolondro** [comment]
It looks like this is caused by a slightly naive implementation of `isWindows` in the Utilities class added [a few days ago](https://github.com/rollbar/rollbar-php/commit/1fa7040e04ad0b0be058ef9de55581dfb344fd94)

```php
return strpos(strtolower(php_uname('s')), 'win') !== false;
```

As Macs returns "Darwin" from this, it's piping the branch to a file called `NUL` on macs instead of /dev/null

**Comment by ArturMoczulski** [comment]
Fixed in tag v1.6.2 https://github.com/rollbar/rollbar-php/releases/tag/v1.6.2
